apps/mark-scan/frontend/src/lib/assistive_technology.ts

```diff
@@ -18,7 +18,10 @@ export function handleKeyboardEvent(event: KeyboardEvent): void {
       triggerPageNavigationButton(PageNavigationButtonId.PREVIOUS);
       break;
     case Keybinding.PAGE_NEXT:
-      triggerPageNavigationButton(PageNavigationButtonId.NEXT);
+      triggerPageNavigationButton(
```

Comment:
Worth making the same change to https://github.com/votingworks/vxsuite/blob/main/apps/mark/frontend/src/lib/gamepad.ts#L28 just so that instance of the "print my ballot" button isn't in a broken state on VxMarkOld